sdk/translation/azure-ai-translation-document/README.md

```diff
@@ -86,6 +86,35 @@ credential = AzureKeyCredential("<api_key>")
 document_translation_client = DocumentTranslationClient(endpoint, credential)
 ```
 
+#### Create the client with an Azure Active Directory credential
+
+`AzureKeyCredential` authentication is used in the examples in this getting started guide, but you can also
+authenticate with Azure Active Directory using the [azure-identity][azure_identity] library.
+Note that regional endpoints do not support AAD authentication. Create a [custom subdomain][custom_subdomain]
+name for your resource in order to use this type of authentication.
+
+To use the [DefaultAzureCredential][default_azure_credential] type shown below, or other credential types provided
+with the Azure SDK, please install the `azure-identity` package:
+
+```pip install azure-identity```
+
+You will also need to [register a new AAD application and grant access][register_aad_app] to
+Form Recognizer by assigning the `"Cognitive Services User"` role to your service principal.
+
+Once completed, set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables:
+`AZURE_CLIENT_ID`, `AZURE_TENANT_ID`, `AZURE_CLIENT_SECRET`.
+
+```python
+from azure.identity import DefaultAzureCredential
+from azure.ai.translation.document import DocumentTranslationClient
+credential = DefaultAzureCredential()
+
+document_translation_client = DocumentTranslationClient(
+    endpoint="https://<my-custom-subdomain>.cognitiveservices.azure.com/",
```

Comment:
in the above examples, when talking about endpoint the reference is to resource name and not the subdomain. 
to maintain consistency, consider using the same everywhere.
i.e. `endpoint = "https://<resource-name>.cognitiveservices.azure.com/"`

Reply:
Good point, making that change!